src/window.cpp

```diff
@@ -1944,6 +1945,7 @@ void ResetWindowSystem()
 static void DecreaseWindowCounters()
 {
 	if (_scroller_click_timeout != 0) _scroller_click_timeout--;
+	if (_hundredth_tick_timeout != 0) _hundredth_tick_timeout--;
```

Comment:
DecreaseWindowCounters() is called via a GUITimer, so I think OnHundredthTick is called as often as before.

Reply:
Actually, nevermind. If OnHundredthTick() stalls the game, _hundredth_tick_timeout won't continue. So there is still 99 DecreaseWindowCounters-ticks before the next invocation.